content/en/docs/instrumentation/js/exporters.md

```diff
@@ -7,10 +7,34 @@ In order to visualize and analyze your traces and metrics, you will need to expo
 
 Below you will find some introductions on how to setup backends and the matching exporters.
 
+- [OTLP endpoint or Collector](#otlp-endpoint-or-collector)
 - [Jaeger](#jaeger)
 - [Zipkin](#zipkin)
 - [Prometheus](#prometheus)
-- [OpenTelemetry Collector](#opentelemetry-collector)
+
+## OTLP endpoint or Collector
+
+To send trace data to a Collector you'll want to use an exporter package (e.g., `@opentelemetry/exporter-trace-otlp-http`):
```

Comment:
Let's avoid using "e.g." in the docs (yes I know there are other occurrences, but let's avoid introducing new ones).

Maybe:

> To send trace data ... package, such as `@opentelemetry/exporter-trace-otlp-http`.